book-worm-2010/book-worm-fe
# Feature/form style responsiveness

### What does this PR do? 

Update styling components for accessibility and responsiveness.  Add error handling for book search and include date formatting for better user readability.

### Is this a feature or Fix/Refactor?  

Feature and refactor

### Problems Encountered & Resulting Solutions  

There was a looping problem that was happening in Home.js that we discovered was due to syntax error in our useEffect function-- when we added the missing brackets, it stopped the continuous rendering. 

One resulting issue is the larger Celebration animation/ component is frozen.  It still renders on submission of a finished book but the elements are not moving as they should be.  This will be a later refactor since it doesn't disrupt any other functionality.

### Where should the reviewer start?

All scss files, Home.js, NewBookForm.js, BookDetails.js, and NewBookmarkForm.js

### How should this be tested?

Run `npm start` in your terminal in the project directory to open the browser and go to localhost:3000 to login with Google Auth and explore the page.  Please click on all of the links to ensure proper functionality.  

### Steps To Take
